### ISSUE TITLE
Was making this a bad idea?

### ISSUE THREAD

**Issue by Remimstr** [Remimstr/COVID-19-globeview#9]
These articles have me questioning whether I should have ever made this.

https://medium.com/nightingale/ten-considerations-before-you-create-another-chart-about-covid-19-27d3bd691be8
https://www.esri.com/arcgis-blog/products/product/mapping/mapping-coronavirus-responsibly/?fbclid=IwAR1l_T73emqkZhsWFOpr0DtE86aKec7Opd8QXrnhrfhqWLXA79WdV_vPdwY

As someone who believes in science and the importance of doing it well, I'm considering shutting this dashboard down. The alternative is to avoid all of the pitfalls listed in the articles.

**Comment by hugetiny** [comment]
I think you're thinking too much about it, treating it like a game instead of a homework assignment for the teacher

**Comment by Remimstr** [comment]
I think there are some steps that we can take to make this a little more responsible:
1. Change the color scheme to blue (done)
2. Add a disclaimer that this was just a cool software project and not medically backed
3. Investigate the data source for reputability / accuracy
4. Plot the data against population (absolute numbers instead of relative ones) - may require finding a new data source

**Comment by hugetiny** [comment]
Require a new data source is difficult.No one could ensure the accuracy of data.
We are hired from no one and do this project for no purpose.
Just,don't think too much...

**Comment by Remimstr** [comment]
Yeah you're right, I was a little nervous because I shared this on facebook and reddit and got 10k+ views. I didn't want it to incite panic if it spread too much but now that views are leveling off, I'm less worried.
